lib/prepArticles.js

```diff
@@ -41,18 +37,18 @@ function prepAll(store, category, callback) {
             	// Only send these elements to frontend, we don't need the rest of the article's details yet
                 let newArticle = {title:article.title, urlToImage:article.urlToImage, snippet:article.content, url:article.url}
 
-            	scrape(article.url , text => {
+            	scrape(article.url , parsedResult => {
 		            completedArticles++
-		            if (text){
-		            	console.log("Done Scraping: " + completedArticles + " / " + articleList.length + "\t(" + category + ")")
+		            
+					console.log("Done Scraping: " + completedArticles + " / " + articleList.length + "\t(" + category + ")")
 
-		            	newArticle.text = text
-		            	newArticle.timeToRead = Math.ceil(text.split(" ").length / 265)	// Average adult reading speed
-		            	newArticle.sentiment = sentiment(text)
-		            	result.push(newArticle)
-		            }
+					newArticle.text = parsedResult.content
+					newArticle.timeToRead = Math.ceil(parsedResult.word_count / AVG_WORDS_PER_MINUTE)
+					newArticle.sentiment = sentiment(parsedResult.content)
+					result.push(newArticle)
+		            
 
-		            if (completedArticles === articleList.length) {
+		            if (completedArticles >= articleList.length) {
```

Comment:
Reason?

Reply:
I thought I addressed this before lol, it's to account for glitches where article list length randomly exceeds the max

Reply:
Oh right on sounds good 👌